odoo/tools/image.py

```diff
@@ -3,18 +3,13 @@
 import base64
 import binascii
 import io
-from typing import Tuple, Union
-
-from PIL import Image, ImageOps
-# We can preload Ico too because it is considered safe
-from PIL import IcoImagePlugin
-try:
-    from PIL.Image import Transpose, Palette, Resampling
-except ImportError:
-    Transpose = Palette = Resampling = Image
 
+from typing import Union
```

Comment:
Since you removed Tuple, you can remove Union too (which is just `A | B`)

Reply:
I removed Tuple because the CI was complaining, but ok 👌🏻